src/neuroconv/tools/nwb_helpers/_dataset_configuration.py

```diff
@@ -172,3 +172,82 @@ def get_default_dataset_io_configurations(
                 )
 
                 yield dataset_io_configuration
+
+
+def get_existing_dataset_io_configurations(
+    nwbfile: NWBFile,
+    backend: Literal["hdf5", "zarr"],
```

Comment:
maybe we could make the default that you use the same backend as the original file